backend/shared/shared/azure_adfs/auth.py

```diff
@@ -148,6 +169,13 @@ def get_graph_api_access_token(self, access_token):
         return response_json["access_token"]
 
     def assign_local_groups(self, user):
+        # Set group membership which tells if user was logged in using ADFS
+        adfs_login_group, _ = Group.objects.get_or_create(name=adfs_login_group_name())
+        if user.is_authenticated:
+            user.groups.add(adfs_login_group)
+        else:
+            user.groups.remove(adfs_login_group)
```

Comment:
You could put the same type of comment `# safety - shouldn't be ever needed` as on line 188

Reply:
or remove else clause on both cases

Reply:
Changed the comments ([#1](https://github.com/City-of-Helsinki/yjdh/pull/1615/commits/19e97161dffc6d3ff38cebfaac0122a447c7af35#diff-7bf3fa8dc236385602f1b56075d236b632c9a655d62aba3756e6146520598e40R181), [#2](https://github.com/City-of-Helsinki/yjdh/pull/1615/commits/19e97161dffc6d3ff38cebfaac0122a447c7af35#diff-7bf3fa8dc236385602f1b56075d236b632c9a655d62aba3756e6146520598e40L160-R193)) to be more in line with reality AFAIK.